ctapipe_io_lst/__init__.py

```diff
@@ -102,6 +102,38 @@ def __init__(self, **kwargs):
     def rewind(self):
         self.multi_file.rewind()
 
+    def subarray(self):
+        """
+        Obtain the subarray from the EventSource
+        Returns
+        -------
+        ctapipe.instrument.SubarrayDecription
+        """
+
+        tel_id = 0
```

Comment:
the `tel_id` for LST1 is 1

Reply:
This is what I mean:

```In [1]: import ctapipe                                                                                               

In [2]: ctapipe.__version__                                                                                          
Out[2]: '0.7.0.post118+gitc513869'

In [3]: import ctapipe_io_lst                                                                                        

In [4]: ctapipe_io_lst.LSTEventSource()                                                                              
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-4-a79cc889ecf4> in <module>
----> 1 ctapipe_io_lst.LSTEventSource()

/usr/local/lib/python3.6/dist-packages/traitlets/traitlets.py in __new__(cls, *args, **kwargs)
    953         new_meth = super(HasDescriptors, cls).__new__
    954         if new_meth is object.__new__:
--> 955             inst = new_meth(cls)
    956         else:
    957             inst = new_meth(cls, *args, **kwargs)

TypeError: Can't instantiate abstract class LSTEventSource with abstract methods subarray
```

I've just changed `tel_id` to 1.

Reply:
Then this is not a problem with ctapipe v0.7 but with the master version of ctapipe.